website/docs/cdktf/python/r/cloud9_environment_ec2.html.markdown

```diff
@@ -30,7 +30,8 @@ class MyConvertedCode(TerraformStack):
         super().__init__(scope, name)
         Cloud9EnvironmentEc2(self, "example",
             instance_type="t2.micro",
-            name="example-env"
+            name="example-env",
```

Comment:
I manually changed this and the equivalent from a few `cdktf` files for now so it doesn't get lost.  I think I need to run something to re-generate these files, but I'm not sure what it is?

Reply:
The CDKTF documentation periodically gets generated from the `website/docs/[d|r]` files by a GitHub action, it does not need to be manually edited (we are going to update the Contributor Guide to capture this).